packages/core/admin/admin/src/content-manager/utils/schema.js

```diff
@@ -55,7 +55,12 @@ const getAttributes = (data) => get(data, ['attributes'], {});
 const createYupSchema = (
   model,
   { components },
-  options = { isCreatingEntry: true, isDraft: true, isFromComponent: false }
+  options = {
+    isCreatingEntry: true,
+    isDraft: true,
+    isFromComponent: false,
+    formatJsonBeforeValidate: false,
```

Comment:
I think it would be nice if we could avoid adding another option. I haven't had time to look but I'm confused why the JSON is stringified in the edit view but still an object in the list view. Do you know why that is the case? IMO it would be better to have consistent data instead of handling different cases.

Reply:
oh sorry missed this:

Run Strapi app using Postgres as DB and try to save this value as a JSON: "{\"a\": 123}", it should keep the string format and not transform it to an object before saving it

so the data from the server is an object not a string?

Reply:
> On the Edit View, the JSON input would stringify that before make the validation
Do you know where this happens and why? 

Sorry not trying to be a bug, I'm just thinking we are going to end up stringifying in two places when it could be done in one place.